packages/js/src/components/TimestampToggle.js

```diff
@@ -0,0 +1,163 @@
+import {Component} from '@wordpress/element';
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import {
+	Toggle, FieldGroup,
+} from '@yoast/components';
+import { __ } from "@wordpress/i18n";
+import popupWindow from '../helpers/popupWindow';
+import {AuthenticationModal} from './modals/wordproof/AuthenticationModal';
+import {handleAPIResponse} from '../helpers/wincherEndpoints';
+import {getSettings} from '../helpers/wordproofEndpoints';
+
+
+const Timestamp = styled.div`
+	display: flex;
+	margin-top: 8px;
+`;
+
+function SettingsLink(props) {
+	const data = window.wordproofSdk.data || {};
+
+	if (!props.isAuthenticated)
+		return( "" );
+
+	return (
+		<a href={data.popup_redirect_settings_url} onClick={( e ) => {
+			e.preventDefault();
+			props.openSettings();
+		}}
+		>{__( 'Manage WordProof settings', 'wordpress-seo' )}</a>
+	);
+
+}
+
+function ToggleWrapper(props) {
+	return(
+		<div onClick={ () => { (!props.isAuthenticated) ? props.openAuthentication() : '' }}>
```

Comment:
`onClick` handlers on non-clickable elements are an accessibility problem. Could you replace this authentication check to a `handleToggle` method in the `TimestampToggle` component:

```js
handleToggle( whatever ) {
  // do authentication here
  this.props.onToggle( whatever );
}
```